packages/suite/src/support/messages.ts

```diff
@@ -3745,6 +3745,42 @@ export default defineMessages({
         id: 'TR_ACCOUNT_TYPE_BIP44_TECH',
         defaultMessage: 'BIP44, P2PKH, Base58',
     },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_NAME: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_NAME',
+        defaultMessage: 'Bip44',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_CHANGE_NAME: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_CHANGE_NAME',
+        defaultMessage: 'Bip44Change',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_ROOT_NAME: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_ROOT_NAME',
+        defaultMessage: 'Bip44Root',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_TECH: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_TECH',
+        defaultMessage: 'BIP44, Base58',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_CHANGE_TECH: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_CHANGE_TECH',
+        defaultMessage: 'BIP44, Base58',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_ROOT_TECH: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_ROOT_TECH',
+        defaultMessage: 'BIP44, Base58',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_DESC: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_DESC',
+        defaultMessage: 'Bip44 account',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_CHANGE_DESC: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_CHANGE_DESC',
+        defaultMessage: 'Bip44Change account',
+    },
+    TR_ACCOUNT_TYPE_SOLANA_BIP44_ROOT_DESC: {
+        id: 'TR_ACCOUNT_TYPE_SOLANA_BIP44_ROOT_DESC',
+        defaultMessage: 'Bip44Root account',
+    },
```

Comment:
I suppose that the product team might have thought about this so I kept these descriptions simple, it can be changed later to whatever is more suitable